spec/session_spec.rb

```diff
@@ -19,6 +20,7 @@ class Session < Binance::Session
       subject { Session.new(options) }
       it 'include the updated options' do
         expect(subject.base_url).to eq(expected_options[:base_url])
+        expect(subject.proxy_url).to eq(expected_options[:proxy_url])
```

Comment:
[SUG] - Creo que en el otro `it` (linea 36) se podría agregar que se espere que `proxy_url` sea `nil`

Reply:
buena